libraries/AP_Filesystem/AP_Filesystem_FlashMemory_LittleFS.cpp

```diff
@@ -810,6 +811,13 @@ uint32_t AP_Filesystem_FlashMemory_LittleFS::find_block_size_and_count() {
     fs_cfg.prog_size = page_size;
     fs_cfg.block_size = flash_block_size * LFS_FLASH_BLOCKS_PER_BLOCK;
     fs_cfg.block_count = flash_block_count / LFS_FLASH_BLOCKS_PER_BLOCK;
+
+    // larger metadata blocks mean less frequent compaction operations, but each
+    // takes longer. however, the total time spent compacting for a given file
+    // size still goes down with larger metadata blocks. 4096 was chosen to
+    // minimize both frequency and log buffer utilization.
+    fs_cfg.metadata_max = 4096;
```

Comment:
what was this before?

Reply:
The block size, i.e. 65536z.

Reply:
I mean I think the lower value is better here, but I recall there being some other default.